src/searchForm/autoComplete/section/AutoCompleteSection.tsx

```diff
@@ -19,6 +21,8 @@ export type AutoCompleteSectionProps = Omit<
   onClose: () => void
 }
 
+const TransitionAutoComplete = styled(TransitionBaseStyle)``
```

Comment:
There is an empty styled component here ☝️

Same in _src/searchForm/datePicker/section/DatePickerSection.tsx_ and _src/searchForm/stepper/section/StepperSection.tsx_

Reply:
Typescript is complaining, it's expecting a: ``